### ISSUE TITLE
No KSCrash.framework built for iOS using Carthage

### ISSUE THREAD

**Issue by ChrisBuchholz** [getsentry/sentry-swift#6]
When I try to install Sentry-Swift using Carthage, no KSCrash.framework is being built for iOS, thus I have been unable to get it working on iOS.

I noticed that KSCrash.framework is being built for Mac twice, so there might be a problem there.

```
*** Building scheme "KSCrash" in Mac.xcworkspace
*** Building scheme "KSCrash-TVOS" in TVOS.xcworkspace
*** Building scheme "KSCrash" in Mac.xcworkspace
```

**Comment by joshdholtz** [comment]
@ChrisBuchholz Looking into right now for ya 👌

**Comment by joshdholtz** [comment]
@ChrisBuchholz This seems to be an issue with how `KSCrash` is building under Carthage 😖

To get around this, you can use `carthage update --platform iOS` for now (which will build the `KSCrash` framework for iOS). I will look into making a PR into `KSCrash` that fixes that Carthage build so that you don't have to specify `platform` 😇

**Comment by ChrisBuchholz** [comment]
It works, thank you very much!

**Comment by joshdholtz** [comment]
@ChrisBuchholz Just wanted to give you an update that my PR for this was merged into KSCrash today - https://github.com/kstenerud/KSCrash/pull/153

I will be fixing the KSCrash dependency in here today so its no fixed here here here yet but it will be soon 😊

**Comment by ChrisBuchholz** [comment]
@joshdholtz: Sounds great :)

**Comment by avocade** [comment]
+1

**Comment by joshdholtz** [comment]
This should be fixed now with the release for `0.2.0`. I will close for now unless seems to magically popup again 😱
